internal/cmd/skupper/token/kube/token_redeem.go

```diff
@@ -0,0 +1,187 @@
+package kube
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/skupperproject/skupper/client"
+	"github.com/skupperproject/skupper/internal/cmd/skupper/utils"
+	"github.com/skupperproject/skupper/pkg/apis/skupper/v1alpha1"
+	"github.com/skupperproject/skupper/pkg/generated/client/clientset/versioned/scheme"
+	skupperv1alpha1 "github.com/skupperproject/skupper/pkg/generated/client/clientset/versioned/typed/skupper/v1alpha1"
+	"github.com/skupperproject/skupper/pkg/utils/validator"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/serializer/json"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	tokenRedeemLong    = "Redeem a token file in order to create a link to a remote site."
+	tokenRedeemExample = "skupper token redeem ~/token1.yaml"
+)
+
+type TokenRedeem struct {
+	timeout time.Duration
+}
+
+type CmdTokenRedeem struct {
+	client     skupperv1alpha1.SkupperV1alpha1Interface
+	CobraCmd   cobra.Command
+	flags      TokenRedeem
+	name       string
+	namespace  string
+	fileName   string
+	KubeClient kubernetes.Interface
+	activeSite *v1alpha1.Site
+}
+
+func NewCmdTokenRedeem() *CmdTokenRedeem {
+
+	skupperCmd := CmdTokenRedeem{}
+
+	cmd := cobra.Command{
+		Use:     "redeem <name>",
```

Comment:
It should be:

`Use: "redeem <filename>"

Reply:
fixed